src/go/intelligence/pipeline/ml_box.go

```diff
@@ -0,0 +1,24 @@
+package pipeline
+
+import (
+	"go/intelligence/models"
+)
+
+// Module     : intelligence
+// Author     : Rahul Indra <indrarahul2013 AT gmail dot com>
+// Created    : Wed, 1 July 2020 11:04:01 GMT
+// Description: CERN MONIT infrastructure Intelligence Module
+
+//MlBox Machine Learning predicted Data
+//To be implemented
+func MlBox(data <-chan models.AmJSON) <-chan models.AmJSON {
+
+	predictedData := make(chan models.AmJSON)
+	go func() {
+		for d := range data {
+			predictedData <- d
```

Comment:
Put doc string here: IMPLEMENT THE LOGIC OF ML PREDICTIONS, and write that so far we just return the data back without any logic

Reply:
Fixed This.